src/app.py

```diff
@@ -1,24 +1,167 @@
-from dash import Dash, html
+import dash
+from dash import html, dcc
+from dash.dependencies import Input, Output
+import plotly.express as px
+import pandas as pd
 
-# Initialize the Dash app
-app = Dash(__name__)
+# Initialize the Dash application
+app = dash.Dash(__name__)
 server = app.server
 
-# Define the layout of the app
-app.layout = html.Div([
-    html.H1("American Housing Prices Dashboard"),
-    html.H2("Project Overview"),
-    html.P("This dashboard presents an interactive visualization of housing prices in top cities and states across top 50 cities in the United States of America. The data is sourced from a comprehensive dataset available on Kaggle, which includes demographics and house prices of the top cities in the United States. Our dashboard is built using the Dash package, enabling users to filter and explore the housing market data across various dimensions, such as state, city, house type, square footage, number of bedrooms and bathrooms, and more."),
-    html.H3("This dashboard is for UBC MDS Program DSCI 532"),
-    html.H4("Authors:"),
-    html.Ul([
-        html.Li("Ruocong Sun"),
-        html.Li("Chun Li"),
-        html.Li("Jake Barnabe"),
-        html.Li("Zeily Garcia")
-    ])
+# Load and prepare the dataset
+df = pd.read_csv('../data/processed/processed_df.csv')
```

Comment:
I think we might have to remove the `../` in front of `data` to make it compatible on render.com.
In short, this line should be:
df = pd.read_csv('../data/processed/processed_df.csv')
The `../data` is required for testing things locally, but when deployed, the default working directory is the root of our repo which means we do not to go one folder down first

Reply:
You're absolutely right @lichunubc 
Will commit that change asap, thank you!